packages/core/admin/ee/server/utils/index.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const { pick, mapKeys } = require('lodash/fp');
+
+const getService = (name) => {
+  return strapi.service(`admin::${name}`);
+};
+
+function mapObject(obj, mapper) {
+  const picked = pick(obj, mapper.pick);
+  return mapKeys(picked, (value, key) => mapper.rename[key] || key);
```

Comment:
```suggestion
  return mapKeys(picked, (value, key) => mapper.rename[key] ?? key);
```

Reply:
:O What is this evilness
Learned something today